tests/commands/test_init.py

```diff
@@ -100,24 +105,23 @@ def test_force(tmp_path, config):
 def test_bad_name(config):
     cmd = InitCommand(config)
     with pytest.raises(CraftError):
-        cmd.run(Namespace(name="1234", author="שראלה ישראל", force=False))
+        cmd.run(Namespace(name="1234", author="שראלה ישראל", force=False, profile=DEFAULT_PROFILE))
```

Comment:
Seems a bit verbose to have to add the `profile` parameter in unrelated tests, doesn't it? Maybe it's just me
It feels a bit awkward to have to pass something that has a default

Reply:
Sure. The only suggestion that comes to me now, if reducing the amount of `profile=DEFAULT_PROFILE` repetition is ever desired, would be to have a `create_parser()` function/fixture that can reuse the `fill_parser()` code that already exists. Then this line could be `cmd.run(parser.parse_args(namespace=Namespace(name="..."))` and the default would be applied
Just some good ol' _Friday bikeshedding_